### PR TITLE
Frontend with full functionality done up by frontend team

### PR DESCRIPTION
**Pull request by mingda2000**
…h on right column still wonky. Changed the colour of buttons on right column as well.

### CODE REVIEW

**Comment by hoofangyu** [comment]
@liauzhanyi could you take a look at how we changed the imports for the params.py in order for our code to work? The part where we need the user to switch working directory to the root inorder to import the functions

File: backend/des/params.py [hunk 1]
```diff
@@ -19,7 +25,8 @@ def get_month_arrival_rate(month : int):
     Returns:
         dict: A dictionary where key is hour (0-23) and values are the mean arrivals for each corresponding time interval.
     """
-    return {h : val for (m, h), val in get_arrival_rates().items() if m == month}
+    return {h : val for (m, h), val in get_arrival_rates(CP_LIST).items() if m == month}
+    return {h : val for (m, h), val in get_arrival_rates(CP_LIST).items() if m == month}
```
Comment (matchastack) [inline backend/des/params.py]: duplicate lines. help delete

File: backend/des/params.py [hunk 2]
```diff
@@ -1,13 +1,19 @@
 import pandas as pd
 import datetime
+import os
 
-CAP_FPATH = "../../data/CP Lots NUS.xlsx"
-DATA_FPATH = "../../data/Cleaned/all_carparks_cleaned.csv"
+path = os.getcwd()
+CAP_FPATH = path + r"\data\CP Lots NUS.xlsx"
```
Comment (matchastack) [inline backend/des/params.py]: `CAP_FPATH = os.path.join(path, "data", "CP Lots NUS.xlsx")`

File: backend/des/params.py [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
+DATA_FPATH = path + r"\data\cleaned\all_carparks_cleaned.csv"
```
Comment (matchastack) [inline backend/des/params.py]: `DATA_FPATH = os.path.join(path, "data", "cleaned", "all_carparks_cleaned.csv")`

File: frontend/pages/parkapp.py [hunk 4]
```diff
@@ -1,31 +1,52 @@
...
 import random
+import time
+import io
+from dash_extensions.enrich import FileSystemCache, Trigger
+
+import os
+path = os.getcwd()
+
+import sys
+sys.path.append(path + r'\backend\des')
```
Comment (matchastack) [inline frontend/pages/parkapp.py]: `sys.path.append(os.path.join(path, "backend", "des"))`

**Review by matchastack — CHANGES_REQUESTED** [review]
replace all path concatenation with `os.path.join`. Difference in OS conventions will cause path error. 

@hoofangyu Works on mine with the changes I suggested, see if it works for you too

**Comment by hoofangyu** [comment]
All good @liauzhanyi

**Review by hoofangyu — APPROVED** [review]
helper functions ok

**Review by matchastack — APPROVED** [review]
looks good!

**Pull request merged**